docs/source/Plugin/P093.rst

```diff
@@ -21,16 +21,33 @@ Maintainer: |P093_maintainer|
 
 Used libraries: |P093_usedlibraries|
 
+Plugin is based on [Arduino library to control Mitsubishi Heat Pumps](https://github.com/SwiCago/HeatPump).
```

Comment:
the rst files don't support Markdown syntax, as they are Sphinx 

See: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html

If you know a way to make Sphinx use Markdown, please let me know as I really don't like the complex syntax of adding links. That is way easier in Markdown.

Reply:
If you are using PlatformIO + VS code you can open a terminal and go to the docs dir.
Then execute the make of your platform (make.bat on Windows) with `html` as parameter.
![image](https://user-images.githubusercontent.com/3751318/76092534-d549d780-5fbf-11ea-89fd-672d7f74e02f.png)